hazelcast/src/test/java/com/hazelcast/test/annotation/IntegrationTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+/**
+ * Tests which should be executed after a final JAR is packaged.
+ * These tests are executed by the Fail-Safe Maven plugin.
+ * (other tests are executed by the Surefire plugin)
+ *
+ */
+public final class IntegrationTest {
```

Comment:
I can try it, but the category is easier, since I can use fixed strings :) It's already magical enough :D
```bash
find . -type f -name *.java -exec sed -i 's/@Category(NightlyTest.class)/@Category(NightlyTest.class)\n@org.junit.Ignore/g' {} +
find . -type f -name *.java -exec sed -i 's/@Category({NightlyTest.class, ParallelTest.class})/@Category({NightlyTest.class, ParallelTest.class})\n@org.junit.Ignore/g' {} +
```